FlowCrypt/Functionality/Services/Compose Message Service/ComposeMessageService.swift

```diff
@@ -111,18 +111,18 @@ final class ComposeMessageService {
             }
     }
 
-    private func getPubKeys(for recepients: [ComposeMessageRecipient]) -> Result<[String], MessageValidationError> {
-        let pubKeys = recepients.map {
-            ($0.email, contactsService.retrievePubKey(for: $0.email))
+    private func getPubKeys(for recipients: [ComposeMessageRecipient]) -> Result<[String], MessageValidationError> {
+        let pubKeys = recipients.map {
+            ($0.email, contactsService.retrievePubKeys(for: $0.email))
         }
 
-        let emailsWithoutPubKeys = pubKeys.filter { $0.1 == nil }.map(\.0)
+        let emailsWithoutPubKeys = pubKeys.filter { $0.1.isEmpty }.map(\.0)
```

Comment:
This multiple-depth reference is getting a bit confusing, could it use a named argument? At least the first one?